packages/uniforms-antd/__tests__/index.ts

```diff
@@ -39,15 +39,17 @@ describe('@RTL', () => {
   suites.testErrorField(theme.ErrorField);
   suites.testErrorsField(theme.ErrorsField);
   suites.testHiddenField(theme.HiddenField);
-  suites.testListDelField(theme.ListDelField);
   suites.testListAddField(theme.ListAddField);
+  suites.testListDelField(theme.ListDelField);
   suites.testListField(theme.ListField, {
     getListAddField: screen => screen.getByRole('img', { name: 'plus-square' }),
   });
   suites.testLongTextField(theme.LongTextField);
   // FIXME: AntD number input doesn't work with new RTL test implementation
   // suites.testNumField(antd.NumField);
   suites.testQuickForm(theme.QuickForm);
+  // FIXME: AntD radio.group does not support data-attributes https://github.com/ant-design/ant-design/issues/8561
+  // suites.testRadioField(theme.RadioField);
```

Comment:
If some of the `RadioField` tests work, then please use them, e.g., by adding a `ignoreDataAttributes` flag. (IIRC that was the sole problem.)

If not, then restore the previous test suite.